engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/PrivateConstructorAnalysis.java

```diff
@@ -74,6 +74,7 @@ public Module runModule(Module ir, ModuleContext moduleContext) {
         ir.imports(),
         ir.exports(),
         newBindings,
+        ir.isPrivate(),
```

Comment:
This is probably not related to _copying itself_, but a fix for some newly discovered omission, right? CCing @Akirathan

Reply:
Yes